front-end/public/index.html

```diff
@@ -29,6 +29,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div id = "belowRoot"></div>
```

Comment:
Was this manually added? What is this for?

Reply:
Feel free to put anything like footers into the App.tsx file

Reply:
@raysarinas yes it was manually added. I added this because I did not want to cover the search bar when showing the result of the search. @raysarinas @ChrisChrisLoLo is this alright? or is there a better way to do it?

Reply:
I think the best way to render the result is to put it into a Page component and put the Page into our router in `App.tsx`. It might be useful to cross-compare the other React pages we currently have to see how this is done

Reply:
I'll do that.